src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +75,36 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    public Block getBlockNum() {
```

Comment:
Is there a reason for adding the getter methods?
If there isn't please remove the unnecessary methods.